### PR TITLE
i18n(es): translate `dialog`

### PR DESCRIPTION
**Pull request by dreyfus92**
#### What kind of changes does this PR include?

- Translated content

#### Description

- What does this PR change? Give us a brief description. 
Translated `dialog.mdx` to spanish

### CODE REVIEW

File: src/content/docs/es/features/dialog.mdx [hunk 1]
```diff
@@ -0,0 +1,302 @@
...
+    ```rust
+    #[cfg_attr(mobile, tauri::mobile_entry_point)]
+    pub fn run() {
+        tauri::Builder::default()
+            // Inicializa el plugin
+            .plugin(tauri_plugin_dialog::init())
+            .run(tauri::generate_context!())
+            .expect("error while running tauri application");
+    }
+    ```
```
Comment (marc2332) [inline src/content/docs/es/features/dialog.mdx]: This is already on L48

File: src/content/docs/es/features/dialog.mdx [hunk 2]
```diff
@@ -0,0 +1,302 @@
...
+Los diálogos nativos del sistema para abrir y guardar archivos junto con diálogos de mensajes.
+
+## Configuración
+
+Instala el plugin de diálogo para comenzar.
+
+<Tabs>
+    <TabItem label="Automatic">
+
+    1. Usa el gestor de paquetes de tu proyecto para agregar la dependencia:
```
Comment (marc2332) [inline src/content/docs/es/features/dialog.mdx]:
```suggestion
    Usa el gestor de paquetes de tu proyecto para agregar la dependencia:
```

File: src/content/docs/es/features/dialog.mdx [hunk 3]
```diff
@@ -0,0 +1,302 @@
...
+        yarn="yarn add @tauri-apps/plugin-dialog"
+        pnpm="pnpm add @tauri-apps/plugin-dialog"
+    />
+
+</TabItem>
+</Tabs>
+
+## Uso
+
+El diálogo está disponible tanto en JavaScript como en Rust. Aquí tienes cómo puedes usarlo:
```
Comment (marc2332) [inline src/content/docs/es/features/dialog.mdx]:
```suggestion
El plugin de diálogo está disponible tanto en JavaScript como en Rust. Aquí tienes cómo puedes usarlo:
```

File: src/content/docs/es/features/dialog.mdx [hunk 4]
```diff
@@ -0,0 +1,302 @@
...
+// regresa un `Option` de file_path, o `None` si el usuario cierra el diálogo
+```
+
+Si necesitas una operación no bloqueante puedes usar `show()` en su lugar:
+
+```rust
+use tauri_plugin_dialog::DialogExt;
+
+app.dialog().file().pick_file(|file_path| {
+    // regresa un `Option` de file_path, o `None` si el usuario cierra el diálogo
```
Comment (marc2332) [inline src/content/docs/es/features/dialog.mdx]:
```suggestion
    // devuelve un `Option` de file_path, o `None` si el usuario cierra el diálogo
```

File: src/content/docs/es/features/dialog.mdx [hunk 5]
```diff
@@ -0,0 +1,302 @@
...
+
+```rust
+use tauri_plugin_dialog::DialogExt;
+
+let file_path = app
+    .dialog()
+    .file()
+    .add_filter("My Filter", &["png", "jpeg"])
+    .blocking_save_file();
+    // has algo con la ruta de archivo opcional aquí
```
Comment (marc2332) [inline src/content/docs/es/features/dialog.mdx]:
```suggestion
    // haz algo con la ruta de archivo opcional aquí
```

File: src/content/docs/es/features/dialog.mdx [hunk 6]
```diff
@@ -0,0 +1,302 @@
...
+o, alternativamente:
+
+```rust
+use tauri_plugin_dialog::DialogExt;
+
+app.dialog()
+    .file()
+    .add_filter("My Filter", &["png", "jpeg"])
+    .pick_file(|file_path| {
+        // regresa un `Option` de file_path, o `None` si el usuario cierra el diálogo
```
Comment (marc2332) [inline src/content/docs/es/features/dialog.mdx]:
```suggestion
        // devuelve un `Option` de file_path, o `None` si el usuario cierra el diálogo
```

File: src/content/docs/es/features/dialog.mdx [hunk 7]
```diff
@@ -0,0 +1,302 @@
...
+
+#### Construir un diálogo de selección de archivos
+
+#### Seleccionar archivos
+
+```rust
+use tauri_plugin_dialog::DialogExt;
+
+let file_path = app.dialog().file().blocking_pick_file();
+// regresa un `Option` de file_path, o `None` si el usuario cierra el diálogo
```
Comment (marc2332) [inline src/content/docs/es/features/dialog.mdx]:
```suggestion
// devuelve un `Option` de file_path, o `None` si el usuario cierra el diálogo
```

**Review by marc2332 — CHANGES_REQUESTED** [review]

**Comment by dreyfus92** [comment]
Thank you for the review @marc2332 🫡

**Review by marc2332 — APPROVED** [review]

**Pull request merged**